extensions/debank/src/index.tsx

```diff
@@ -0,0 +1,322 @@
+import {
+  ActionPanel,
+  ActionPanelItem,
+  AlertActionStyle,
+  Color,
+  confirmAlert,
+  Form,
+  Icon,
+  List,
+  ListSection,
+  OpenInBrowserAction,
+  SubmitFormAction,
+  useNavigation,
+} from "@raycast/api";
+import { ChainId, ComplexProtocol, Token } from "@yukaii/debank-types";
+import { useEffect, useMemo, useState } from "react";
+import { getComplexProtocolList, getTokenList, getTotalBalance, TotalBalance } from "./api";
+import useFavoriteAddresses from "./utils/useFavoriteAddresses";
+
+export default function Command() {
+  const [address, setAddress] = useState("");
+  const onChange = (value: string) => setAddress(value);
+  const { push } = useNavigation();
+
+  const title = useMemo(() => {
+    if (!address) {
+      return "Enter Wallet Address";
+    } else {
+      return "Press Enter to Search";
+    }
+  }, [address]);
+
+  const { loading, favoriteAddresses, addFavoriteAddress, removeFavoriteAddress } = useFavoriteAddresses();
+  const { pop } = useNavigation();
+
+  const defaultItem = (
+    <List.Item
+      title={title}
+      actions={
+        address && (
+          <ActionPanel>
+            <ActionPanelItem
+              title={`Show Balance`}
+              icon={Icon.Binoculars}
+              onAction={() => {
+                push(<BalanceView address={address} />);
+              }}
+            />
+            <OpenInBrowserAction url={`https://debank.com/profile/${address}`} title="Open on DeBank" />
+            {!favoriteAddresses.find((add) => address && add.address === address) && (
+              <ActionPanelItem
+                title={`Add to Favorites`}
+                icon={Icon.Star}
+                shortcut={{ modifiers: ["cmd", "shift"], key: "f" }}
+                onAction={() => {
+                  push(
+                    <AddFavoriteAddressForm
+                      address={address}
+                      onSubmit={({ address, identifier }) => {
+                        addFavoriteAddress(address, identifier);
+                        pop();
+                      }}
+                    />
+                  );
+                }}
+              />
+            )}
+          </ActionPanel>
+        )
+      }
+    />
+  );
+
+  return (
+    <List onSearchTextChange={onChange}>
```

Comment:
What do you think about setting the `isLoading` prop when you are loading the address?